sql_generators/glean_usage/templates/macros.sql

```diff
@@ -27,5 +27,7 @@ _core_clients_first_seen AS (
   JOIN
      _core
   ON _fennec_id_lookup.fennec_client_id = _core.client_id
+  QUALIFY
+    ROW_NUMBER() OVER (PARTITION BY client_id ORDER BY first_seen_date ASC) = 1
```

Comment:
CI is failing because zetasql requires `QUALIFY` to be used in conjunction with `WHERE` or `HAVING`.

also, per [this comment](https://github.com/mozilla/bigquery-etl/pull/3201/files#r961907636), this should throw an error if `client_id` (old or new) is not unique:

```suggestion
  WHERE
    TRUE
  QUALIFY
    IF(
      COUNT(*) OVER (PARTITION BY _fennec_id_lookup.client_id) > 1
      OR COUNT(*) OVER (PARTITION BY _core.client_id) > 1,
      ERROR("duplicate client_id detected"),
      TRUE
    )
```